src/core/Akka/Util/Base64Encoding.cs

```diff
@@ -24,14 +25,40 @@ public static class Base64Encoding
         /// </summary>
         /// <param name="value">TBD</param>
         /// <returns>TBD</returns>
-        public static string Base64Encode(this long value) => Base64Encode(value, new StringBuilder()).ToString();
+        public static string Base64Encode(this long value)
+        {
+            return Base64Encode(value, "");
+        }
+
+        public static string Base64Encode(this long value, string prefix)
+        {
+            // 11 is the number of characters it takes to represent long.MaxValue
+            // so we will never need a larger size for encoding longs
+            Span<char> sb = stackalloc char[11 + prefix?.Length ?? 0];
+            var spanIndex = 0;
+            if (!string.IsNullOrEmpty(prefix) && prefix.Length > 0)
```

Comment:
Is there a reason for checking both IsNullOrEmpty as well as length on the prefix? 

Also, would `.IsNullOrWhiteSpace` be a better choice? (idk?)

Reply:
> Is there a reason for checking both IsNullOrEmpty as well as length on the prefix?

I thought it was easier to read than using the null-coalescing operator

Reply:
Changed to `IsNullOrWhiteSpace` since the latter is illegal and we already check for `string.Empty` with the length check.